src/Umbraco.Web.UI/Umbraco.Web.UI.csproj

```diff
@@ -1973,6 +1973,8 @@
     <Content Include="Views\Partials\Grid\Editors\Textstring.cshtml" />
     <Content Include="Views\Partials\Grid\Bootstrap2.cshtml" />
     <Content Include="Views\Partials\Grid\Editors\Base.cshtml" />
+    <Content Include="Views\Partials\Grid\Bootstrap3-Fluid.cshtml" />

```

Comment:
I'm not really sure why they were never there in the first place. We ship with them, so they should be included in the csproj as well as the other grid renderers.